File: app/src/main/java/com/commit451/nativestackblur/sample/MainActivity.java
package com.commit451.nativestackblur.sample;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.widget.ImageView;

import com.squareup.picasso.Picasso;

public class MainActivity extends AppCompatActivity {

    ImageView mImage;
    ImageView mBlurredImage;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        mImage = (ImageView) findViewById(R.id.image);
        mBlurredImage = (ImageView) findViewById(R.id.blurredImage);
        Picasso.with(this)
                .load("http://www.androidcentral.com/sites/androidcentral.com/files/styles/large/public/article_images/2014/10/lollipop-statue-2.jpg?itok=RC1ovcEz")
                .into(mImage);

        Picasso.with(this)
                .load("http://www.androidcentral.com/sites/androidcentral.com/files/styles/large/public/article_images/2014/10/lollipop-statue-2.jpg?itok=RC1ovcEz")
                .transform(new BlurTransformation(8))
                .into(mBlurredImage);
    }
}

File: app/src/main/java/com/commit451/nativestackblur/sample/MainActivity.kt
package com.commit451.nativestackblur.sample

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.ImageView
import coil.load

class MainActivity : AppCompatActivity() {

    companion object {
        private const val ImageUrl =
            "https://pbs.twimg.com/profile_images/1164525925242986497/N5_DCXYQ_400x400.jpg"
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val image = findViewById<ImageView>(R.id.image)
        val blurredImage = findViewById<ImageView>(R.id.blurredImage)
        image.load(ImageUrl)
        blurredImage.load(ImageUrl) {
            transformations(BlurTransformation())
        }
    }
}